src/components/List/StatusIcon.js

```diff
@@ -0,0 +1,39 @@
+import { CircularLoader, IconErrorFilled16 } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+import css from './List.module.css'
+
+export const ErrorIcon = ({ numberOfErrors }) => {
+    return (
+        <span className={css.errorIcon}>
+            <IconErrorFilled16 color="#B71C1C" />
```

Comment:
you should also be able to import colors from @dhis2/ui and then define it as color={colors.red700}